coralproject/talk
# [CSS Fix] HR line on older devices - Flexbox

## What does this PR do?

Fixes the positioning of the `hr` in the Delete account modal in some older devices including:
-Mac Sierra(Safari)
-Nexus9(Tablet)_Firefox
-Android_v8_Firefox

![Coral Talk](https://image.ibb.co/geVMO8/Screen_Shot_2018_05_23_at_5_43_26_PM.png)


## How do I test this PR?
- Load Coral Talk in Safari on macOS Sierra (version 10.12) 
- Click Delete My Account in Settings
- See the `hr` in modal positioned correctly and not as in screenshot
